_scripts/check.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+if [[ $(which docker) && $(docker --version) ]]; then
+  docker=y
+else
+  docker=n
+fi
+
+os="$(uname -a | cut -f 1 -d ' ')"
+if [ "$os" = "Darwin" ]; then
+    if [ "$docker" = "n" ]; then
+      echo "install docker to continue installation"
+      echo "https://docs.docker.com/docker-for-mac/install/"
+      exit 1
+    fi
```

Comment:
let's add a check for node and npm

then we can remove this bit `Note: Please use Node.js 8+.` in the README

Reply:
plus can we try to validate that node is version is `8` and npm is `5.8 or higher` (don't worry about this if that is too complex)

Reply:
In the new node checking section we can put cut this part of the readme https://github.com/deeptibaghel/fxa-local-dev/blob/3a12d3ec253df1a745f652c31e5725e829b5c4d4/README.md#os-x and print it out. After we add it here we can simplify the readme.
 
if npm is old we can tell people to upgrade `Run the following "npm i -g npm@5"

Reply:
We need to do this one for sure and be detailed with the required version. For example `5.0.0`, is no good, but `5.8.0` is good. By the time we land this we should probably require some version of npm 6 though

Reply:
> In the new node checking section we can put cut this part of the readme

@vladikoff this one is not very clear to me, other points are done i think except for the merge conflict which i will handle in next commit.

Reply:
> @vladikoff this one is not very clear to me, other points are done i think except for the merge conflict which i will handle in next commit.

Basically **move** the text from the README.md into `_scripts/check_node_version.js`.  In that script detect if it is running in OS X and print out that section of the readme.